src/pages/Home.jsx

```diff
@@ -4,8 +4,14 @@ function Home() {
   return (
     <div>
       <Header/>
-    Home
-    </div>
+      <div style={{ maxWidth: '1600px' }} className='mx-auto mt-10 ml-20 mr-20'>
+      <div className='flex mx-auto flex-col items-center justify-center '>
+        <p className='text-3xl'>Hi, this is the best news site, better than BBC</p>
+        <p className='text-xl'>In order to get acces to functionality of this web site you</p>
+        <p className='text-xl'>Have to be <span className='font-medium'>login</span></p>
```

Comment:
It might be nice to change this to a link to direct the user to the login page? 🤔